src/libs/export.c

```diff
@@ -1232,6 +1232,13 @@ void gui_init(dt_lib_module_t *self)
 
   d->format = dt_bauhaus_combobox_new_action(DT_ACTION(self));
   dt_bauhaus_widget_set_label(d->format, NULL, N_("file format"));
+  gtk_widget_set_tooltip_text(GTK_WIDGET(d->format),
+    _("images will be exported according to the format specified here\n\n"
+      "when exporting to AVIF, EXR, JPEG XL, or XCF, selecting specific\n"
+      "metadata is not currently possible\n\n"
+      "for these formats, any metadata fields will not be included\n"
```

Comment:
I think "no metadata fields will be included" would be better.
Consider `gtk_widget_set_tooltip_markup` to allow "...selects \<b\>all\</b\> of the metadata..."

Reply:
> I think "no metadata fields will be included" would be better.

Indeed.

> Consider gtk_widget_set_tooltip_markup to allow "...selects <b>all</b> of the metadata..."

Agree. Thanks, also done.